### PR TITLE
Club: adding missing pallette colors

### PR DESCRIPTION
**Pull request by matiasbenedetto**
#### Changes proposed in this Pull Request:
Club: adding missing pallette colors

image

### CODE REVIEW

File: club/theme.json [hunk 1]
```diff
@@ -46,6 +46,16 @@
 					"slug": "background",
 					"color": "#F6EB80",
 					"name": "Background"
+				},
+                {
```
Comment (jffng) [inline club/theme.json:40]: Some awkward formatting going on here?

File: club/theme.json [hunk 2]
```diff
@@ -46,6 +46,16 @@
...
+					"slug": "primary",
+					"color": "#000AFF",
```
Comment (jffng) [inline club/theme.json]: What is the benefit to having this twice (both foreground and primary)? I guess if the user wants to change the button and link colors for example but not the foreground color?

File: club/theme.json [hunk 3]
```diff
@@ -46,6 +46,16 @@
...
+					"name": "Primary"
+				},
+				{
+					"slug": "tetriary",
```
Comment (jffng) [inline club/theme.json]: Kind of a nitpick, but `tertiary` is technically third order I believe, so should this actually be `secondary`?

File: club/theme.json (hunk 2 above)
Comment (matiasbenedetto) [reply club/theme.json]: I think the design is adding this option with the idea of making it customizable.

File: club/theme.json (hunk 3 above)
Comment (matiasbenedetto) [reply club/theme.json]: :thinking: I saw this in many of our latest themes. Tertiary color without a secondary one. I think the name can be related to the color theory and not to the number of colors in our palette. Primary is, of course, the primary color, and background would be the secondary.

File: club/theme.json (hunk 1 above)
Comment (matiasbenedetto) [reply club/theme.json:40]: Seems to be fixed now

File: club/theme.json (hunk 3 above)
Comment (scruffian) [reply club/theme.json]: I think that made more sense when we used the customizer for colors, but I don't know that it makes sense in block themes.

File: club/theme.json [hunk 4]
```diff
@@ -37,17 +37,27 @@
...
+                {
+                    "slug": "primary",
+                    "color": "#000AFF",
+                    "name": "Primary"
+                },
+                {
+                    "slug": "tetriary",
+                    "color": "#FFF7A7",
+                    "name": "Tetriary"
+                }
```
Comment (vcanales) [inline club/theme.json]: 👋  nothing to add other than this small typo:

```suggestion
                {
                    "slug": "tertiary",
                    "color": "#FFF7A7",
                    "name": "Tertiary"
                }
```

**Review by jffng — APPROVED** [review]

**Pull request merged**